pulseapi/entries/test_views.py

```diff
@@ -620,12 +630,38 @@ def test_entry_serializer(self):
             # Make sure that the property exists in the serialized entry
             self.assertIn('creators_with_profiles', serialized_entry)
             creators_with_profiles = serialized_entry['creators_with_profiles']
+            self.assertIn('related_creators', serialized_entry)
+            related_creators = serialized_entry['related_creators']
             # Make sure that the number of serialized creators matches the
             # number of creators for that entry in the db
-            self.assertEqual(
-                len(creators_with_profiles),
-                OrderedCreatorRecord.objects.filter(entry=entry).count()
-            )
+            db_entry_creator_count = OrderedCreatorRecord.objects.filter(entry=entry).count()
+            self.assertEqual(len(creators_with_profiles), db_entry_creator_count)
+            self.assertEqual(len(related_creators), db_entry_creator_count)
+
+    def test_post_entry_related_creators(self):
+        """
+        Make sure that we can post related creators with an entry
+        """
+        creator1_id = self.creators[0].id
+        creator2_name = 'Pomax'
```

Comment:
I know it's tempting, but please let's not put "currently employee" names in a test suite that can easily outlive someone's employment. It's... really iffy to have to write a patch that changes that person's name to something else later down the line.